### PR TITLE
Adding some of DW's projects

### PR DESCRIPTION
**Pull request by davidwyld**
Still a couple more to do.

### CODE REVIEW

File: _data/projects.csv [hunk 1]
```diff
@@ -35,4 +35,5 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
 65,,Hole in the Road,Hole in the Road,,Matt Leach,24/09/2020,13/10/2020,Computer Science,2,1
 71,,JBC: Data and Data Science,JBC: Data and Data Science,,"David Jones, David Wilby",01/01/2021,01/07/2021,Department of Health and Social Care,2,1
 75,airqo,AirQo: Air Quality (Google),AirQo: Kampala Air Quality,"python, scrum, testing, machine learning, Gaussian processes, containers, cloud, continuous integration",Robert Turner,01/03/2021,01/05/2021,Computer Science,1,1
-78,,Rail Transport,Rail Transport,,"John Charlton, Peter Heywood",29/01/2021,28/11/2021,Mechanical Engineering,2,1
\ No newline at end of file
+78,,Rail Transport,Rail Transport,,"John Charlton, Peter Heywood",29/01/2021,28/11/2021,Mechanical Engineering,2,1
+,firstdraftcoviddash,FirstDraft x GATE COVID Dashboard,FirstDraft x GATE COVID Data Deficits Dashboard,"python, django, vue, javascript, web apps, elasticsearch, postgres, databases","David Wilby, Twin Karmakharm",01/06/2020,31/07/2020,Computer Science,1,1
```
Comment (drj11) [inline _data/projects.csv]: Should there be a numeric ID in column 1?

Should `x` be `×` (U+00B7 MULTIPLICATION SIGN)?

**Review by drj11 — APPROVED** [review]
Merge now and no another PR for the other projects.

File: _data/projects.csv (hunk 1 above)
Comment (davidwyld) [reply _data/projects.csv]: nope, it's fine to have an empty ID, ID's come from RSEadmin.

Yes, you're right on the × - this is what I get for being lazy.

File: _data/projects.csv [hunk 2]
```diff
@@ -17,17 +17,17 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
...
 25,,A Network For Lemons,A Network For Lemons,,Anna Krystalli,09/09/2019,08/09/2020,Management School,2,1
-27,,IMPACT Acceleration,IMPACT Acceleration,,David Wilby,01/08/2019,31/07/2021,Computer Science,2,1
+27,comsciimpact,IMPACT Acceleration,IMPACT Acceleration,,David Wilby,01/08/2019,31/07/2021,Computer Science,1,1
 28,itshpc,IT Services HPC Support 2019-2021,IT Services HPC Support 2019-2021,"HPC, Slurm, Puppet, Nagios, collectd, Grafana, InfluxDB, SGE, Grid Engine, Singularity",Will Furnass,01/08/2019,01/08/2021,IT Services,1,1
 30,,Accelerate,Accelerate,,"Peter Heywood, Mozhgan Kabiri Chimeh, Matt Leach",01/03/2016,01/10/2021,Computer Science,2,1
 31,,Teaching Support for GPU Module,Teaching Support for GPU Module,,"Mozhgan Kabiri Chimeh, Robert Chisholm",01/02/2019,01/09/2021,Computer Science,2,1
 32,,SoBigData++,SoBigData++,,"David Jones, Twin Karmakharm",01/01/2020,01/01/2022,Computer Science,2,1
-33,,RISIS II (includes WeVerfiy and some ELG),RISIS II (includes WeVerfiy and some ELG),,"Twin Karmakharm, David Wilby",01/01/2020,30/06/2021,Computer Science,2,1
+33,risis2,RISIS II,RISIS II,,"David Jones, David Wilby",01/01/2020,30/06/2021,Computer Science,2,1
```
Comment (willfurnass) [inline _data/projects.csv]: Need to decrement 2 to 1?

File: _project_descriptions/comsciimpact.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+**GATEcloud usage statistics**
+
+Few research software projects measure their own usage, but data demonstrating impact can be essential to gaining future funding for software. GATE is a software framework for text analytics, developed by the Natural Language Processing group. A large number of analytic pipelines are available via API calls to the widely-used [GATEcloud](https://cloud.gate.ac.uk/) service.
+
+The RSE team worked with GATE to build a simple tool to regularly analyse the usage of these services and make up to date statistics available internally for use in funding applications and reports.
+
+
+**DeepQuest**
+
+A neural network-based tool for quality estimation in machine translation. The RSE team worked with researchers to refactor a proof of principle codebase which demonstrated excellent results in using deep learning to estimate the quality of linguistic translations. The impact of the project was enhanced by refactoring the codebase in order to create a python package, developing command line  and python interfaces, reducing the size of the codebase and adding tests and documentation.
```
Comment (willfurnass) [inline _project_descriptions/comsciimpact.md]:
```suggestion
A neural network-based tool for quality estimation in machine translation. The RSE team worked with researchers to refactor a proof of principle codebase which demonstrated excellent results in using deep learning to estimate the quality of linguistic translations. The impact of the project was enhanced by refactoring the codebase in order to create a python package, developing command line  and Python interfaces, reducing the size of the codebase and adding tests and documentation.
```

File: _project_descriptions/firstdraftcoviddash.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+key: firstdraftcoviddash
+---
+
+During the initial wave of the COVID-19 pandemic in 2020, [First Draft](http://firstdraftnews.org/) worked with researchers in the [GATE team](http://gate.ac.uk/) to analyse the availability and demand of fact checking in various areas of misinformation relating to COVID-19.
+
+The RSE team worked with GATE RSEs to build a [data visualisation dashboard](https://datadeficits.firstdraftnews.org/) to provide data visualisation and analysis for this research.
+
+The dashboard calculates and visualises the geographic spread of availability of fact checking vs demand on different topics of misinformation. As well as providing interactive components showing indicators for information demand and information supply.
+
+It was developed in Django with a front-end in Vue JS.
```
Comment (willfurnass) [inline _project_descriptions/firstdraftcoviddash.md]:
```suggestion
It was developed using the Django web framework with a front-end implemented in Vue.js.
```

**Review by willfurnass — APPROVED** [review]
See minor comments

File: _data/projects.csv (hunk 2 above)
Comment (davidwyld) [reply _data/projects.csv]: well spotted, thanks

**Pull request merged**